test/test_suite.py

```diff
@@ -136,3 +143,94 @@ def test_matplotlib_plot_invalid_json(
         )
         assert main.execute(args) == 1
         mock_plot_save.assert_not_called()
+
+
+class TestLS(unittest.TestCase):
+    def setUp(self) -> None:
+        """Create a temporary directory and test files."""
+        self.temp_dir = tempfile.TemporaryDirectory()
+        self.files = {
+            "file1_info.json": {
+                "schema_version": MINIMUM_SCHEMA_VERSION,
+                "prefix": "test1",
+            },
+            "file2_info.json": {
+                "schema_version": MINIMUM_SCHEMA_VERSION,
+                "prefix": "test2",
+            },
+            "file3_info.json": {"schema_version": "0.1.0", "prefix": "old_version"},
+            "not_matching.json": {
+                "schema_version": MINIMUM_SCHEMA_VERSION,
+                "prefix": "no_match",
+            },
+        }
+        for filename, content in self.files.items():
+            with open(os.path.join(self.temp_dir.name, filename), "w") as f:
+                json.dump(content, f)
+
+    def tearDown(self) -> None:
+        """Clean up the temporary directory."""
+        self.temp_dir.cleanup()
+
+    def _run_ls(self, pattern: str, fmt: str) -> str:
+        """Helper function to run ls() and capture stdout."""
+        args = argparse.Namespace(
+            pattern=[os.path.join(self.temp_dir.name, pattern)],
+            colors=False,
+            fields=["prefix", "schema_version"],
+            format=fmt,
+            func=ls,
+        )
+        buf = StringIO()
+        with contextlib.redirect_stdout(buf):
+            exit_code = ls(args)
+            self.assertEqual(exit_code, 0)
+        return buf.getvalue().strip()
+
+    def test_ls_sanity(self) -> None:
+        """Basic sanity test to ensure ls() runs without crashing."""
+        result = self._run_ls("file*", "summaries")
+
+        self.assertIn("Prefix:", result)
+        prefixes = [
+            line.split(":", 1)[1].strip()
+            for line in result.splitlines()
+            if line.startswith("Prefix:")
+        ]
+        # There should be exactly 2 prefixes (file1 and file2)
+        self.assertEqual(len(prefixes), 2)
+        self.assertTrue(any("file1" in p for p in prefixes))
+        self.assertTrue(any("file2" in p for p in prefixes))
+        # file3 does not meet minimum schema version
+        self.assertNotIn("file3", result)
+        # not_matching does not meet the glob pattern
+        self.assertNotIn("not_matching", result)
```

Comment:
make it consistent since you already use pytest, so smth like

```suggestion
        assert len(prefixes) == 2
        assert any("file1" in p for p in prefixes)
        assert any("file2" in p for p in prefixes)
        # file3 does not meet minimum schema version
        assert "file3" not in result
        # not_matching does not meet the glob pattern
        assert "not_matching" not in result
```